packages/pluggableWidgets/switch-web/src/ui/switch-preview.scss

```diff
@@ -0,0 +1,21 @@
+@import "mixins";
```

Comment:
Nice approach of doing these preview styles! Atlas isn't necessary to have these different brand colors in place.

Reply:
Thanks. If Atlas is present, end-users can configure the Style design-property, however, the styles present in Design mode environment (in Studio Pro) come from the local styles to the widget. I'm not sure if this is the same in Studio or if the Atlas_Core styles are present, too.

Reply:
Is that the only reason why you added them to the widget? Studio Pro has/had an issue where it doesn't apply class names from custom design properties, like those shipped with Atlas.

Reply:
Yes, it was the reason for adding them in. I think they should remain, despite the Studio Pro issue you mention. Like you said, projects with this widget without Atlas 3 should still be preview-able in Design mode.

Reply:
I ended up changing the widget's local preview styles (`getPreviewCss`) to exclude the brand styling that exists in Atlas. Now, the preview in Design Mode will rely on Atlas styling for the different brands in preview mode. I can't remember why, but when I first worked on the preview mode, Atlas styling wasn't present in the Design Mode environment. Anyways, this is much cleaner.